Xamarin.Forms.Platform.UAP/Shapes/ShapeRenderer.cs

```diff
@@ -74,15 +74,15 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 #endif	
 		void UpdateHeight()
 		{
-			Control.Height = Element.Height;
+			Control.Height = Element.Height > 0 ? Element.Height : 0;
```

Comment:
I'd use `Math.Max(Element.Height, 0);` but that's personal preference I guess

Reply:
I like the suggestion, changed.